CHANGELOG.md

```diff
@@ -1,13 +1,25 @@
+## [UNRELEASED] neptune-optuna 1.0.0
```

Comment:
Why? I don't think it's a hard breaking change update :wink:. Let's only bump the minor.